django_project/base/models/project.py

```diff
@@ -208,3 +215,14 @@ def pagination_threshold_exceeded(self):
             return True
         else:
             return False
+
+
+class ProjectScreenshots(models.Model):
+    """Model to store multiple screenshots for each project."""
```

Comment:
"A model to store a screenshot linked to a project."

Maybe we should have a sort order fields as undoubtedly someone is going to want to specify the order in which they display on the carousel....